test/remote/gateways/remote_litle_test.rb

```diff
@@ -231,6 +231,42 @@ def test_authorization_capture_refund_void
     assert_equal 'Approved', void.message
   end
 
+  def test_authorization_capture_with_stored_credential
```

Comment:
Minor quibble, we usually refer to this as `authorize_and_capture`, partly not to confuse it with a response's `authorization`.